themes/src/main/resources/theme/base/admin/messages/admin-messages_pt_BR.properties

```diff
@@ -0,0 +1,814 @@
+#encoding: utf-8
+consoleTitle=Console de administração do Keycloak
+
+# Common messages
+enabled=Habilitado
+name=Nome
+displayName=Nome de exibição
+displayNameHtml=Nome de exibição HTML
+save=Salvar
+cancel=Cancelar
+onText=Sim
+offText=Não
+client=Cliente
+clients=Clientes
+clear=Limpar
+selectOne=Selecione um...
```

Comment:
I would ask you to consider the same name convention in use by {{admin-messages_en.properties}}. For instance, instead of "Selecione um..." we should have "Selecione Um...". Considering that {{{admin-messages_en.properties}} defines "Select One..." (with upper O) and not "Select one...".